hushline/templates/directory.html

```diff
@@ -1,80 +1,42 @@
 {% extends "base.html" %} {% block title %}Directory{% endblock %} {% block
 content %}
 <h2>User Directory</h2>
-{% if directory_verified_tab_enabled %} {% if not logged_in %}
-<p class="helper section-intro">
-  👋
-  <a href="https://hushline.app" target="_blank">Hush Line</a>
-  connects whistleblowers with lawyers, journalists, business leaders, and more.
-  We advise speaking to a legal professional before taking any confidential
-  information from your workplace.
-</p>
+{% if intro_text %}
+  <p class="helper section-intro">{{ intro_text | safe }}</p>
```

Comment:
We shouldn't use the `safe` filter. The output of the sanitizer should be immediately wrapped in a `Markup` so that we get (semi-) type safety and can't accidentally fuck up later and have a plain `str` marked as `safe`.

Reply:
✅